e2e_playwright/shared/toolbar_utils.py

```diff
@@ -54,14 +54,14 @@ def assert_fullscreen_toolbar_button_interactions(
     # Check that it is visible
     assert_snapshot(
         fullscreen_wrapper,
-        name=f"{filename_prefix if filename_prefix != "" else widget_test_id}-fullscreen_expanded",
+        name=f"{filename_prefix if filename_prefix != '' else widget_test_id}-fullscreen_expanded",
```

Comment:
nit: the PR describes changes in preparation to Vite, but this is a Python file. Totally fine for me to make the fixes here 🙂 just wanted to sanity check that this is what you intended to update

Reply:
Ah. Yes. So I cannot explain why my changes caused the issue but the E2E tests failed for this issue.

f"{filename_prefix if filename_prefix != "" else widget_test_id}-fullscreen_collapsed"

```
name=f"{filename_prefix if filename_prefix != "" else widget_test_id}-fullscreen_expanded",
                                                                                              ^
SyntaxError: f-string: expecting '}'
```

Seeing the issue, it seems like we cannot have double quotes inside an interpolated string. I don't know how it worked in the first place

Reply:
Wild! But yeah, makes sense 😁